mmdet/models/detectors/test_mixins.py

```diff
@@ -1,11 +1,33 @@
+import logging
+import sys
+
 import torch
 
 from mmdet.core import (bbox2roi, bbox_mapping, merge_aug_bboxes,
                         merge_aug_masks, merge_aug_proposals, multiclass_nms)
 
+logger = logging.getLogger(__name__)
+
+if sys.version_info >= (3, 7):
+    from mmdet.utils.contextmanagers import completed
+
 
 class RPNTestMixin(object):
 
+    if sys.version_info >= (3, 7):
+
+        async def async_test_rpn(self, x, img_meta, rpn_test_cfg):
+            sleep_interval = rpn_test_cfg.pop("async_sleep_interval", 0.025)
```

Comment:
How are the values of `sleep_interval` set? I noticed that 0.025, 0.017 and 0.035 are used.

Reply:
I've measured GPU time required for operations using temporary `profile_time` decorator and set `sleep_interval` a bit higher. I used K80 GPU and single image for measurements.